src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime-project.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Yep, or I could just get rid of this completely, which is my plan.

As far as just using direct dependencies: yes I am well aware. I am planning to do that using a tool across all  of corefx in a future change.  Right now I am just trying to get this working and extra dependencies aren't getting in the way. Trying to lump that work in just further delays this which folks have been needing for months now.